arcgis-ios-sdk-samples/Scenes/Get elevation at a point/GetElevationPointViewController.swift

```diff
@@ -0,0 +1,132 @@
+        super.viewWillTransition(to: size, with: coordinator)
+        dismiss(animated: false)
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        // Add the source code button item to the right of navigation bar.
+        (self.navigationItem.rightBarButtonItem as! SourceCodeBarButtonItem).filenames = ["GetElevationPointViewController", "ElevationViewController"]
+    }
+}
+
+// MARK: - AGSGeoViewTouchDelegate
+extension GetElevationPointViewController: AGSGeoViewTouchDelegate {
+    func geoView(_ geoView: AGSGeoView, didTapAtScreenPoint screenPoint: CGPoint, mapPoint: AGSPoint) {
+        if let relativeSurfacePoint = sceneView?.screen(toBaseSurface: screenPoint) {
+            dismiss(animated: false)
+            
+            // Get the tapped point
+            let point = AGSPoint(x: relativeSurfacePoint.x, y: relativeSurfacePoint.y, spatialReference: .wgs84())
+            if let graphic = graphicsOverlay.graphics.firstObject as? AGSGraphic {
+                // Move the symbol to the tapped point.
+                graphic.geometry = point
+            } else {
+                // Create the symbol at the tapped point.
+                let marker = AGSSimpleMarkerSceneSymbol(style: .sphere, color: .red, height: 100, width: 100, depth: 200, anchorPosition: .center)
+                let graphic = AGSGraphic(geometry: point, symbol: marker)
+                graphicsOverlay.graphics.add(graphic)
+            }
+            
+            // Get the surface elevation at the surface point.
+            self.sceneView.scene?.baseSurface!.elevation(for: relativeSurfacePoint) { (results: Double, error: Error?) in
+                if let error = error {
+                    self.presentAlert(error: error)
+                } else {
+                    self.showPopover(elevation: results, popoverPoint: screenPoint)
+                }
+            }
+        }
+        sceneView.viewpointChangedHandler = { [weak self] in
+            DispatchQueue.main.async {
+                self?.dismiss(animated: false)
+            }
+        }
+    }
+
+    private func showPopover(elevation: Double, popoverPoint: CGPoint) {
+        guard let controller = storyboard?.instantiateViewController(withIdentifier: "ElevationViewController") as? ElevationViewController else {
+                return
+        }
+        // Setup the controller to display as a popover.
+        controller.modalPresentationStyle = .popover
+        controller.loadViewIfNeeded()
+        let elevationMeasurement = Measurement(value: elevation, unit: UnitLength.meters)
+        controller.elevationLabel?.text? = " Elevation at tapped point: " + MeasurementFormatter().string(from: elevationMeasurement)
+        controller.popoverPresentationController?.delegate = self
+        controller.popoverPresentationController?.backgroundColor = .lightGray
```

Comment:
I feel that basic white and black callout is more legible than the grey and white combo.

![IMG_6569E1246717-1](https://user-images.githubusercontent.com/8692125/64463141-c7baa000-d0b7-11e9-968e-10f496256e87.jpeg)

![IMG_0821F22DDE41-1](https://user-images.githubusercontent.com/8692125/64463143-cb4e2700-d0b7-11e9-81c5-dd11a8c28c66.jpeg)

What do you think @philium ?

Reply:
I concur.